### PR TITLE
basics concept

### PR DESCRIPTION
**Pull request by colinleach**
Replaces #253 with something much simpler, after merging #202. Mostly `concepts/basics`, but `introduction.md` is also copied into `lasagna`.

### CODE REVIEW

**Review by jonmcalder — APPROVED** [review]
Thanks @colinleach! Happy to accept this as is but have made a few suggestions for consideration and possible improvements.

File: concepts/basics/about.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+8 %/% 3 # 2 (integer division)
+8 %% 3  # 2 (remainder)
+2 ^ 3   # 8 (exponential)
+```
+
+Also, note that `#` starts a single-line comment. There is no separate syntax for multiline comments.
+
+## Functions
+
+To define a function:
```
Comment (jonmcalder) [inline concepts/basics/about.md:39]: I think a little more elaboration is needed here on how to create a function. If nothing else, it is worth mentioning that functions can have zero or more arguments, separated by commas (and maybe also that arguments can have default values?).

File: concepts/basics/about.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+```
+
+Parentheses are needed with `function()` even if there are no parameters. The braces `{ }` are not strictly required here, but including them is considered good style.
+
+R will automatically return the final value of a function. More generally, `return(x * x)` will work from anywhere in the function body.
+
+Calling a function is conventional:
+
+```R
+y <- squareit(3) # y is now 9
```
[the remark this answers is not in the document]
Comment (colinleach) [reply concepts/basics/about.md:54]: changed

**Comment by colinleach** [comment]
I agree with all your suggestions and have reworked `about.md`. The internal link to `conditionals` will be broken for now, so I'll need to check it later when the relevant concepts are merged.

I'll go ahead and merge this one for now. As the key first concept, I won't be at all surprised if it changes again in future - good, that's why we want multiple pairs of eyes on the documents.

**Pull request merged**